packages/zapp/console/src/components/Executions/Tables/utils.ts

```diff
@@ -4,3 +4,8 @@ import { nameColumnLeftMarginGridWidth } from './styles';
 export function calculateNodeExecutionRowLeftSpacing(level: number, spacing: Spacing) {
   return spacing(nameColumnLeftMarginGridWidth + 3 * level);
 }
+
+export function selectExecution(state, nodeExecution) {
+  // use null in case if there is no execution provied - to close panel
+  state.setSelectedExecution(nodeExecution?.id ?? null);
+}
```

Comment:
I'm not sure it makes sense to add a helper function for such type of call. I think it would be cleaner to just call state.setSelectedExecution(nodeExecution?.id ?? null) in place. Comment about why null is used could leave near `setSelectedExecution` definition if you are trying to avoid it's duplication.

Anyway, if you want to stick with a helper function -> please provide parameters types you are expecting, as:
```
export function selectExecution(state: NodeExecutionsTableState, nodeExecution: NodeExecution) { 
...
}
```

Reply:
agree with that since the logic is pretty simple.